### PR TITLE
Adds list states

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: iot/api/Program.cs [hunk 1]
```diff
@@ -617,6 +633,32 @@ public static object GetDeviceConfigurations(string projectId, string cloudRegio
         }
         // [END iot_get_device_configurations]
 
+        // [START iot_get_device_states]
+        public static object GetDeviceStates(string projectId, string cloudRegion, string registryId, string deviceId)
+        {
+            var cloudIot = CreateAuthorizedClient();
+            // The resource name of the location associated with the key rings.
```
Comment (simonz130) [inline iot/api/Program.cs]: nit: add an empty line between line 639 and the comment

File: iot/api/Program.cs [hunk 2]
```diff
@@ -833,6 +875,7 @@ public static void Main(string[] args)
                 DeleteDeviceOptions,
                 GetDeviceOptions,
                 GetDeviceConfigsOptions,
+                //GetDeviceStatesOptions,
```
Comment (simonz130) [inline iot/api/Program.cs]: Remove commented out line from your code

File: iot/api/Program.cs [hunk 3]
```diff
@@ -851,6 +894,7 @@ public static void Main(string[] args)
                 (DeleteDeviceOptions opts) => DeleteDevice(opts.projectId, opts.regionId, opts.registryId, opts.deviceId),
                 (GetDeviceOptions opts) => GetDevice(opts.projectId, opts.regionId, opts.registryId, opts.deviceId),
                 (GetDeviceConfigsOptions opts) => GetDeviceConfigurations(opts.projectId, opts.regionId, opts.registryId, opts.deviceId),
+                //(GetDeviceStatesOptions opts) => GetDeviceStates(opts.projectId, opts.regionId, opts.registryId, opts.deviceId),
```
Comment (simonz130) [inline iot/api/Program.cs]: ditto

File: iot/api/Program.cs (hunk 1 above)
Comment (gguuss) [reply iot/api/Program.cs]: Done

File: iot/api/Program.cs (hunk 2 above)
Comment (gguuss) [reply iot/api/Program.cs]: Done

File: iot/api/Program.cs (hunk 3 above)
Comment (gguuss) [reply iot/api/Program.cs]: Done

**Pull request merged**